app/uk/gov/hmrc/bindingtariffclassification/model/Event.scala

```diff
@@ -53,11 +59,34 @@ case class CaseStatusChange
   override val from: CaseStatus,
   override val to: CaseStatus,
   override val comment: Option[String] = None,
-  attachmentId: Option[String] = None
-) extends FieldChange[CaseStatus] {
+  override val attachmentId: Option[String] = None
+) extends FieldChange[CaseStatus] with OptionalAttachment {
   override val `type`: EventType.Value = EventType.CASE_STATUS_CHANGE
 }
 
+case class CancellationCaseStatusChange
+(
+  override val from: CaseStatus,
+  override val comment: Option[String] = None,
+  override val attachmentId: Option[String] = None,
+  reason: CancelReason
```

Comment:
Note this will also be on the decision object. Added here too for consistency.

Reply:
It could work as a log as well  ;-). Looks good to me